README.md

```diff
@@ -10,22 +10,25 @@ depending if your chart type already exists more steps may be required. They are
 
 Prereq: Your data already exists in the yaml config file. If not then please implement it in https://github.com/qbicsoftware/statistics-data-retrieval
 
-1. Ensure that the following classes are synced with the eponymic classes from data retrieval tool:
-    * ChartNames
-    * Potentially other lexica enum classes
-    * model.data.ChartConfig
-    * model.data.ChartSetting
-    * model.data.MainConfig
+1. Ensure that the submodule classes are up-to-date:
+    
+    ``git submodule foreach git pull origin master``
```

Comment:
We should include that users have to initialize the submodule
git submodule update --init